.size-snapshot.json

```diff
@@ -1,123 +1,81 @@
 {
   "bundle.js": {
-    "bundled": 823371,
-    "minified": 754553,
-    "gzipped": 91244
+    "bundled": 820258,
```

Comment:
This shakes up the `dist` build artifact since the icons included in Component logic now reside in `dist/bundle-es/icons/icons` instead of `dist/bundle-es/svgs`, so the file ordering got jumbled around and it's hard to track changes. It's always nice to see a smaller bundle, though 🥳